docs/markdown/Contributing.md

```diff
@@ -18,6 +18,25 @@ Github](https://github.com/mesonbuild/meson/pulls). This causes them
 to be run through the CI system. All submissions must pass a full CI
 test run before they are even considered for submission.
 
+## Acceptance and merging
+
+The kind of review and acceptance any merge proposal gets depends on
+the changes it contains.
+
+Documentation only changes can be accepted by any member of the
```

Comment:
Might be worth clarifying if this is `doc/` subdir only or fixes of embedded help strings, comments in code, etc are accepted.